frontend/assets/css/prime.scss

```diff
@@ -248,14 +248,16 @@
   .p-toast-message
   .p-toast-message-content
   .p-toast-message-icon.p-icon {
-  width: 2rem;
-  height: 2rem;
+  width: 1rem;
```

Comment:
:smile: imo `rem` would be the way to go here, but everywhere are `px`-values :man_shrugging:

Reply:
it was `rem` before so i would leave it here also.